packages/balancer/skills/autonomous_fund_abci/models.py

```diff
@@ -108,7 +108,7 @@ def setup(self) -> None:
 class Params(
     LiquidityProvisionParams,
     PoolManagerParams,
-    TransactionParams,
```

Comment:
Why did you remove these? Because already defined in Termination?

Reply:
Yes, its a bit awkward having to do this. Maybe it makes more sense to remove the tx settlement params from the termination params.

Reply:
Although then not setting tx settlement params means you will get the error when the termination tx is submitted.

Reply:
Let's keep monitoring this and act once we have a convincing and transparent design